models/twilio__message_enhanced.sql

```diff
@@ -48,14 +48,14 @@ final as (
         messages.num_segments,
         messages.price,
         messages.price_unit,
-        messages.updated_at,
+        messages.updated_at
 
         {% if var('using_twilio_messaging_service', True) %}
-        messaging_service.friendly_name,
-        messaging_service.inbound_method,
-        messaging_service.us_app_to_person_registered,
-        messaging_service.use_inbound_webhook_on_number,
-        messaging_service.use_case
+        , messaging_service.friendly_name
+        , messaging_service.inbound_method
+        , messaging_service.us_app_to_person_registered
+        , messaging_service.use_inbound_webhook_on_number
+        , messaging_service.use_case
```

Comment:
If the messaging service is disabled, the query will fail due to the trailing comma. This has been fixed with the above update.

Reply:
Thanks Joe for catching this, I must've not ran with the variables disabled. Just as an extension, I added an additional dbt run, with variable to false, in integration testing to both source and transforms.

Reply:
Oh my bad-- just realized you added it already 👍